### PR TITLE
Correct typos in the source docblocks

### PR DESCRIPTION
**Pull request by cdamken**
Correct typos in the source docblocks

### CODE REVIEW

**Comment by cdamken** [comment]
@settermjd could you check?

File: apps/files_external/3rdparty/Dropbox/Exception/RequestToken.php [hunk 1]
```diff
@@ -10,7 +10,7 @@
  */
 
 /**
- * This exception is thrown when an error occured during the request_token process.
```
Comment (DeepDiver1975) [inline apps/files_external/3rdparty/Dropbox/Exception/RequestToken.php]: :-1: this is 3rd party code - we never manipulate 3rdparty code

File: apps/files_external/3rdparty/google-api-php-client/src/Google/Service/Audit.php [hunk 2]
```diff
@@ -144,9 +144,9 @@ class Google_Service_Audit_Activities_Resource extends Google_Service_Resource
    * @opt_param int maxResults Number of activity records to be shown in each
    * page.
    * @opt_param string eventName Name of the event being queried.
-   * @opt_param string startTime Return events which occured at or after this
```
Comment (DeepDiver1975) [inline apps/files_external/3rdparty/google-api-php-client/src/Google/Service/Audit.php]: :-1: this is 3rd party code - we never manipulate 3rdparty code

File: apps/files_external/3rdparty/google-api-php-client/src/Google/Service/Reports.php [hunk 3]
```diff
@@ -278,7 +278,7 @@ class Google_Service_Reports_Activities_Resource extends Google_Service_Resource
    * be retrieved.
    * @param array $optParams Optional parameters.
    *
-   * @opt_param string startTime Return events which occured at or after this
```
Comment (DeepDiver1975) [inline apps/files_external/3rdparty/google-api-php-client/src/Google/Service/Reports.php]: :-1: this is 3rd party code - we never manipulate 3rdparty code

**Comment by DeepDiver1975** [comment]
:-1: this is 3rd party code - we never manipulate 3rdparty code

**Pull request closed without merging**